scripts/node-16kb.py

```
@@ -0,0 +1,44 @@
+import os
+import zipfile
+import logging
+import hashlib
+import shutil
+
+URL = 'https://github.com/alzhuravlev/nodejs-mobile/releases/download/v18.20.4/nodejs-mobile-android.zip'
+ANDROID_LIBNODE_SO = './node_modules/nodejs-mobile-react-native/android/libnode'
+NODE_ZIP_PATH = 'node-mobile.zip'
+
+def download_large_file(url, output_file):
+    with requests.get(url, stream=True) as response:
```

Ruff (0.14.0)

Probable use of `requests` call without timeout

(S113)